lib/evss/disability_compensation_form/configuration_rating_info.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    # Configuration for the findRatingInfoPID service
+    #
+    class ConfigurationRatingInfo < EVSS::DisabilityCompensationForm::Configuration
+      API_VERSION = Settings.evss.versions.common
+
+      def base_path
+        "#{Settings.evss.url}/wss-common-services-web-#{API_VERSION}/rest/ratingInfoService/#{API_VERSION}"
+      end
```

Comment:
This configuration already lives in the [evss/common_service.rb](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/common_service.rb#L8). Looks to me like you're really creating a new endpoint in an existing service rather than a brand new service altogether.

Can you add to or extend the `evss/common_service` and corresponding configuration rather than duplicate?

Reply:
Thank you for pointing that out...I've gone ahead and updated this to use common_service instead of a new service.